src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -1,5 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>IsRecurringDonations2Enabled__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>This option enables Enhanced Recurring Donations. WARNING: When you enable this option, NPSP changes your existing Recurring Donations records and the Recurring Donations object to match the data model for Enhanced Recurring Donations.</description>
```

Comment:
FYI - The helptext and descriptions were modified again in Meena's. PR. Should be final now.

Reply:
@force2b Requested changes have been made to alphabetize fields and update description/helptext.